restaurants.js

```diff
@@ -0,0 +1,31 @@
+/* 
+This route accesses the Yelp API and sends raw search results from Yelp. 
+You can review the Yelp docs here: https://www.yelp.com/developers/documentation/v3/business_search
+*/
+
+const axios = require('axios');
+
+async function getRestaurants(request, response, next) {
+  try {
+
+    let searchQuery = request.query.searchQuery;
+
+    let url = `https://api.yelp.com/v3/businesses/search?location=${searchQuery}&term=restaurants&radius=1000`;
+
```

Comment:
would be helpful to make a comment saying that the parameter radius is in meters for the yelp api!